pysmt/smtlib/solver.py

```diff
@@ -71,9 +104,13 @@ def set_option(self, name, value):
     def set_logic(self, logic):
         self._send_silent_command(SmtLibCommand(smtcmd.SET_LOGIC, [logic]))
 
+    def _debug(self, msg):
```

Comment:
To save a little bit of time you could change ```_debug``` as follows.

```python
def _debug(self, msg, *format_args):
    if self.options.debug_interaction:
        print(msg % format_args)
```

In this way, we can avoid the construction of the debug string if the debugging is not enabled.